charts/stable/node-red/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: nodered/node-red
   # -- image tag
-  tag: 2.1.4
+  tag: 2.2.2
```

Comment:
```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```

Reply:
Thanks for your review and suggestion. Unfortunately I could not apply it as a commit directly in GitHub (there seems to be a restriction, as it it involves a deleted line). So I manually added 07103d343c22d2c219764790629f5f549ce8d330.